### DISCUSSION TITLE
Dahua's infrared channel rtsp renders the visible light channel number

### DISCUSSION THREAD

**Discussion by zjcsix666** [mpromonet/webrtc-streamer#743] [General]
webrtc-streamer可以通过设置参数进行解决吗

**Comment by mpromonet** [comment]
Bonjour @zjcsix666 

Je vous remercie pour votre message et vous invite à consulter le lien suivant https://github.com/mpromonet/webrtc-streamer/blob/master/.github/CONTRIBUTING.md

Cordialement,
Michel.

**Reply by zjcsix666** [reply]
同一个rtsp视频流在vlc播放器上面是热成像的画面,使用webrtc服务播放是可见光的画面
